mobile/android/android-log-filter.ts

```diff
@@ -29,15 +29,20 @@ export class AndroidLogFilter implements Mobile.IPlatformLogFilter {
 		return data + os.EOL;
 	}
 
-	private getConsoleLogFromLine(lineText: string): any {
-		let acceptedTags = ["chromium", "Web Console", "JS"];
+	private getConsoleLogFromLine(lineText: string, pid: string): any {
+		// filter log line if it does not belong to the current application process id
+		if (lineText.indexOf(pid) < 0) {
```

Comment:
So in case we do not have pid, we'll always return null. But in case we have PID, we'll still apply the incorrect filters?
For iOS Simulator we have similar case, but the logic there is:
- in case there's a pid, find it in the logs and show each line that contains it
- in case we do not have a PID, apply the old filtering

Can't we do the same here?